triggers/daml/Daml/Trigger.daml

```diff
@@ -260,3 +268,109 @@ data TriggerState s = TriggerState
   , commandsInFlight : Map CommandId [Command]
   , nextCommandId : Int
   }
+
+-- | Used to construct an 'ACS' for 'testRule'.
```

Comment:
We should probably export this type but without the constructor. We leak it anyway since we expose `toACS`.

Reply:
Yes, done.